tox.ini

```diff
@@ -28,6 +28,9 @@ commands =
 
 [testenv:py27]
 sitepackages = true
+deps = 
+    mock
```

Comment:
please put `mock` into `requirements.txt`, and don't forget to specify condition for python < 3